services/visual_recognition/v3.js

```diff
@@ -16,7 +16,8 @@
 
 module.exports = function(RED) {
   const SERVICE_IDENTIFIER = 'visual-recognition',
-    VisualRecognitionV3 = require('watson-developer-cloud/visual-recognition/v3'),
+    VisualRecognitionV3 = require('ibm-watson/visual-recognition/v3'),
+    { IamAuthenticator } = require('ibm-watson/auth'),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 4.](https://app.codacy.com/manual/BetaWorks-NodeRED-Watson/node-red-node-watson/pullRequest?prid=4764899)